docs/_sources/about.rst.txt

```diff
@@ -0,0 +1,150 @@
+
+.. math::
+
+    E[d(X)=0 \mid Y=1, g(X)] = E[d(X)=0, Y=1]
+
+Generalized Entropy Index
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Generalized Entropy Index is proposed as a unified individual and group fairness measure in [1]_. With :math:`b_i = \hat{y}_i - y_i + 1`:
+
+.. math::
+
+           \mathcal{E}(\alpha) = \begin{cases}
+              \frac{1}{n \alpha (\alpha-1)}\sum_{i=1}^n\left[\left(\frac{b_i}{\mu}\right)^\alpha - 1\right] &
+              \alpha \ne 0, 1, \\
+              \frac{1}{n}\sum_{i=1}^n\frac{b_{i}}{\mu}\ln\frac{b_{i}}{\mu} & \alpha=1, \\
+            -\frac{1}{n}\sum_{i=1}^n\ln\frac{b_{i}}{\mu},& \alpha=0.
+            \end{cases}
+
+References:
+            .. [1] T. Speicher, H. Heidari, N. Grgic-Hlaca, K. P. Gummadi, A. Singla, A. Weller, and M. B. Zafar,
+             A Unified Approach to Quantifying Algorithmic Unfairness: Measuring Individual and Group Unfairness via
+             Inequality Indices, ACM SIGKDD International Conference on Knowledge Discovery and Data Mining, 2018.
+
+Predictive Equality
+^^^^^^^^^^^^^^^^^^^
+Predictive Equality is defined as the situation when accuracy of decisions is equal across two groups, as measured by false positive rate (FPR).
+
+.. math::
+
+    E[d(X)|Y=0, g(X)] = E[d(X), Y=0]
+
+Statistical Parity
+^^^^^^^^^^^^^^^^^^
+Statistical Parity measures the difference in probabilities of a positive outcome across two groups.
+ 
+.. math::
+
+    P(\hat{Y} = 1 | D = \text{group 1}) - P(\hat{Y} = 1 | D = \text{group 2})
+
+Theil Index
+^^^^^^^^^^^
+Theil Index is the generalized entropy index with :math:`\alpha = 1`. See Generalized Entropy Index.
+
+
+Equalized Odds
+^^^^^^^^^^^^^^
+
+Equalized odds is a bias mitigation technique where subset of decisions of a binary classifier is flipped at uniform random in each of two groups to achieve equality of TPR and FPR across the two groups as proposed in [2]_. This subset rate in each group is learned via constrained optimization.
+
+References:
+            .. [2] Moritz Hardt, Eric Price, and Nathan Srebro. 2016. Equality of opportunity in supervised learning. In Proceedings of the 30th International Conference on Neural Information Processing Systems (NIPS'16).
+
+About Recommenders Metrics
+==========================
+
+Jurity offers various standardized metrics for measuring the recommendation performance.
+While the recommendation systems community agrees on these metrics, the implementations can be different, especially when it comes to edge-cases.
+
+For the definitions below, :math:`A` is the set of actual ratings for users and :math:`P` is the set of predictions / recommendations.
+Each :math:`A_i` and :math:`P_i` represents the list of actual items and list of recommended items, respectively, for a user :math:`i`.
+
+
+Binary Recommender Metrics
```

Comment:
Not sure if docs haven't been updated, but something looks off with heading structure on page.

![image](https://user-images.githubusercontent.com/31479649/108852842-144bb100-75b4-11eb-9aed-cb9f615e833e.png)

Reply:
this should be all set